dssm/network_conf.py

```diff
@@ -151,7 +151,7 @@ def create_conv(context_len, hidden_size, prefix):
         logger.info("create a sequence_conv_pool which context width is 4")
         conv_4 = create_conv(4, self.dnn_dims[1], "cnn")
 
-        return conv_3, conv_4
+        return paddle.layer.concat(input=[conv_3, conv_4])
```

Comment:
- Could you please also help to fix the logging information in line 151:
    ```python
    logger.info("create a sequence_conv_pool whose context width is 4.")
    ```
- Could you please confirm with @ChunweiYan , is `concat` or `addto` here used as his original design.

Reply:
Done